middle_end/flambda2/to_cmm/to_cmm_expr.ml

```diff
@@ -743,8 +751,12 @@ and apply_expr env res apply =
       let wrap, _, res = Env.flush_delayed_lets ~mode:Branching_point env res in
       let cmm, free_vars = wrap (C.cexit cont [call] []) free_vars in
       cmm, free_vars, res
-    | Inline { handler_params; handler_body = body; handler_params_occurrences }
-      -> (
+    | Inline
+        { handler_params;
+          handler_body = body;
+          handler_params_occurrences;
+          inlined_debuginfo
```

Comment:
Maybe handler_body_dbg might be clearer?  (bit unclear at the moment whether this is just part of the debuginfo or all of it, and which expression it applies to...)

Reply:
renamed to `handler_body_inlined_debuginfo`